<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="mainWindowLayout">
   <item row="0" column="0" rowspan="7">
    <widget class="Scene" name="scene" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="imagesGroup">
     <property name="title">
      <string>Imágenes</string>
     </property>
     <layout class="QGridLayout" name="imagesGroupLayout">
      <item row="2" column="0">
       <widget class="QRadioButton" name="allImagesButton">
        <property name="text">
         <string>Todo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="searchImagesLineEdit">
        <property name="placeholderText">
         <string>Buscar cosas y lugares</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="streetViewImagesButton">
        <property name="text">
         <string>Street View</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="searchImagesButton">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="markersGroup">
     <property name="title">
      <string>Marcadores</string>
     </property>
     <layout class="QGridLayout" name="markersGroupLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="detectMarkersCheckBox">
        <property name="text">
         <string>Detectar Markers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="scaleSpinBox">
        <property name="minimum">
         <number>-500</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="xRotationLabel">
        <property name="text">
         <string>Rot. X</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="zRotationLabel">
        <property name="text">
         <string>Rot. Z</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="xTranslationLabel">
        <property name="text">
         <string>Tras. X</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="yTranslationSpinBox">
        <property name="minimum">
         <double>-9.990000000000000</double>
        </property>
        <property name="maximum">
         <double>9.990000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="zTranslationLabel">
        <property name="text">
         <string>Tras. Z</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="yRotationLabel">
        <property name="text">
         <string>Rot. Y</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="xTranslationSpinBox">
        <property name="minimum">
         <double>-9.990000000000000</double>
        </property>
        <property name="maximum">
         <double>9.990000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="zTranslationSpinBox">
        <property name="minimum">
         <double>-9.990000000000000</double>
        </property>
        <property name="maximum">
         <double>9.990000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="yTranslationLabel">
        <property name="text">
         <string>Tras. Y</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="scaleLabel">
        <property name="text">
         <string>Escala</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="descriptionCheckBox">
        <property name="text">
         <string>Descripcion</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="yRotationSpinBox">
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="xRotationSpinBox">
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QSpinBox" name="zRotationSpinBox">
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="rotateCheckBox">
        <property name="text">
         <string>Rotar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QSlider" name="rotationVelocitySlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="rotationVelocityLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Velocidad</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="facesGroup">
     <property name="title">
      <string>Rostros</string>
     </property>
     <layout class="QGridLayout" name="facesGroupLayout">
      <item row="3" column="0">
       <widget class="QCheckBox" name="eraseMouthCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Borrar Boca</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="linesWidthLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Líneas</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="detectFacesCheckBox">
        <property name="text">
         <string>Detectar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSlider" name="linesWidthSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSlider" name="blurMouthSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>32</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="blurMouthLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Blur</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QCheckBox" name="firstFaceCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Recortar primer rostro detectado</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QCheckBox" name="seguirRostroCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Seguir rostro</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="musicGroup">
     <property name="title">
      <string>Música</string>
     </property>
     <layout class="QGridLayout" name="musicGroupLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="searchMusicLineEdit">
        <property name="placeholderText">
         <string>Buscar artistas o canciones</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="searchMusicButton">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QTextEdit" name="messagesTextEdit">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Logo" name="appLogo" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="horizontalLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Scene</class>
   <extends>QWidget</extends>
   <header>scene.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Logo</class>
   <extends>QWidget</extends>
   <header>logo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchMusicLineEdit</tabstop>
  <tabstop>searchMusicButton</tabstop>
  <tabstop>detectMarkersCheckBox</tabstop>
  <tabstop>descriptionCheckBox</tabstop>
  <tabstop>scaleSpinBox</tabstop>
  <tabstop>xTranslationSpinBox</tabstop>
  <tabstop>xRotationSpinBox</tabstop>
  <tabstop>yTranslationSpinBox</tabstop>
  <tabstop>yRotationSpinBox</tabstop>
  <tabstop>zTranslationSpinBox</tabstop>
  <tabstop>zRotationSpinBox</tabstop>
  <tabstop>detectFacesCheckBox</tabstop>
  <tabstop>firstFaceCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
